apps/src/applab/applab.js

```diff
@@ -1319,6 +1323,29 @@ studioApp.reset = function(first) {
   Applab.interpreter = null;
 };
 
+// TODO(dave): remove once channel id is passed in appOptions.
```

Comment:
I plan to merge https://github.com/code-dot-org/code-dot-org/pull/2225 today.  I'm just waiting for @sfilman to do some verification on staging.  Maybe just wait until that change is in, then we won't need this temporary code?